packages/shared/components/MenuItem.svelte

```diff
@@ -1,44 +1,52 @@
 <script lang="ts">
     import { Icon, Text, Spinner, Position, InformationTooltip } from 'shared/components'
+    import { Icon as IconEnum } from '@auxiliary/icon'
+    import { MenuItemVariant } from './enums'
 
-    export let icon: string = ''
-    export let iconProps: Record<string, unknown> = undefined
+    export let icon: IconEnum | undefined = undefined
+    export let iconProps: Record<string, unknown> | undefined = undefined
     export let title: string
-    export let subtitle = ''
-    export let onClick: () => any
-    export let selected = false
-    export let disabled = false
-    export let isLoading = false
-    export let enableTooltipVisible = false
-    export let tooltip: string = undefined
+    export let subtitle: string = ''
+    export let onClick: () => void
+    export let selected: boolean = false
+    export let disabled: boolean = false
+    export let isLoading: boolean = false
+    export let enableTooltipVisible: boolean = false
+    export let tooltip: string | undefined = undefined
+    export let variant: MenuItemVariant = MenuItemVariant.Info
```

Comment:
Typescript strict mode isn't working here, but it would be good to correct the types of the props.

Reply:
Not clear to me what `enableTooltipVisible` is for, but I think it is unused